gateway/api.go

```diff
@@ -2730,6 +2730,33 @@ func (gw *Gateway) validateOAS(next http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+func (gw *Gateway) makeImportedOASTykAPI(next http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		_, oasObj, err := extractOASObjFromReq(r.Body)
+		if err != nil {
+			doJSONWrite(w, http.StatusBadRequest, apiError("Couldn't decode OAS object"))
+			return
+		}
+
+		tykExtensionConfigParams := oas.GetTykExtensionConfigParams(r)
```

Comment:
can you add a nil check here? 
if it's nil, we could call `BuildDefaultTykExtension` with empty `TykExtensionConfigParams{}`